### PR TITLE
bump version

### PR DESCRIPTION
**Pull request by pmeier**
We should probably on merge this after release.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -23,6 +23,8 @@ supported Python versions.
 +==========================+==========================+=================================+
 | ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.8``, ``<=3.10``           |
 +--------------------------+--------------------------+---------------------------------+
+| ``2.0.0``                | ``0.15.0``               | ``>=3.8``, ``<=3.10``           |
```
Comment (oke-aditya) [inline README.rst:26]:
```suggestion
| ``2.0.0``                | ``0.15.0``               | ``>=3.8``, ``<=3.11``           |
```

Are we not supporting python 3.11? As per blog seems yes
https://pytorch.org/blog/deprecation-cuda-python-support/

File: README.rst [hunk 2]
```diff
@@ -23,6 +23,8 @@ supported Python versions.
 +==========================+==========================+=================================+
 | ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.8``, ``<=3.10``           |
```
Comment (pmeier) [inline README.rst:24]:
```suggestion
| ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.8``, ``<=3.11``           |
```

**Review by NicolasHug — APPROVED** [review]
Thanks Philip

File: ios/LibTorchvision.podspec [hunk 3]
```diff
@@ -1,8 +1,8 @@
-pytorch_version = '1.13.0'
+pytorch_version = '2.0.0'
 
 Pod::Spec.new do |s|
     s.name             = 'LibTorchvision'
-    s.version          = '0.14.0'
+    s.version          = '0.15.0'
```
Comment (NicolasHug) [inline ios/LibTorchvision.podspec:5]:
```suggestion
    s.version          = '0.15.1'
```

File: README.rst [hunk 4]
```diff
@@ -21,7 +21,9 @@ supported Python versions.
 +--------------------------+--------------------------+---------------------------------+
 | ``torch``                | ``torchvision``          | ``python``                      |
 +==========================+==========================+=================================+
-| ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.8``, ``<=3.10``           |
+| ``main`` / ``nightly``   | ``main`` / ``nightly``   | ``>=3.8``, ``<=3.11``           |
++--------------------------+--------------------------+---------------------------------+
+| ``2.0.0``                | ``0.15.0``               | ``>=3.8``, ``<=3.11``           |
```
Comment (NicolasHug) [inline README.rst:26]:
```suggestion
| ``2.0.0``                | ``0.15.1``               | ``>=3.8``, ``<=3.11``           |
```

**Review by NicolasHug — APPROVED** [review]

**Pull request merged**